Packs/SafeNet_Trusted_Access/Integrations/SafeNetTrustedAccess/SafeNetTrustedAccess.py

```diff
@@ -0,0 +1,832 @@
+        outputs_key_field=['id', 'userName'],
+        outputs=response
+    )
+
+
+def create_user_sta_command(client: Client, args: Dict[str, Any]) -> CommandResults:
+    """ Function for sta-create-user command. Create new user in the tenant. """
+
+    response = client.create_user_sta(args=args)
+    header_sequence = ["id", "schemaVersionNumber", "userName", "firstName", "lastName", "email", "mobileNumber",
+                       "alias1", "alias2", "custom1", "custom2", "custom3", "address", "city", "state", "country",
+                       "postalCode", "isSynchronized"]
+
+    return CommandResults(
+        readable_output=tableToMarkdown("STA user successfully created :", response, headers=header_sequence,
+                                        headerTransform=pascalToSpace, removeNull=True),
+        outputs_prefix='STA.USER',
+        outputs_key_field=['id', 'userName'],
+        outputs=response
+    )
+
+
+def update_user_sta_command(client: Client, args: Dict[str, Any]) -> CommandResults:
+    """ Function for sta-update-user-info command. Update profile of a specific user. """
+
+    response = client.update_user_sta(args=args)
+    header_sequence = ["id", "schemaVersionNumber", "userName", "firstName", "lastName", "email", "mobileNumber",
+                       "alias1", "alias2", "custom1", "custom2", "custom3", "address", "city", "state", "country",
+                       "postalCode", "isSynchronized"]
+
+    return CommandResults(
+        readable_output=tableToMarkdown("STA user successfully updated:", response,
+                                        headers=header_sequence, headerTransform=pascalToSpace, removeNull=True),
+        outputs_prefix='STA.USER',
+        outputs_key_field=['id', 'userName'],
+        outputs=response
+    )
+
+
+def delete_user_sta_command(client: Client, args: Dict[str, Any]) -> CommandResults:
+    """ Function for sta-delete-user command. Delete user from the tenant. """
+
+    response = client.delete_user_sta(userName=args.get('userName'))
+    return CommandResults(
+        readable_output=f"## STA user - {args.get('userName')} successfully deleted.",
+        outputs_prefix='STA.USER.DELETE',
+        outputs=response.status_code
+    )
+
+
+def get_user_groups_sta_command(client: Client, args: Dict[str, Any]) -> CommandResults:
+    """ Function for sta-get-user-groups command. Get all the groups associated with a specific user. """
+
+    response = client.get_user_groups_sta(userName=args.get('userName'), limit=args.get('limit'))
+    header_sequence = ['id', 'schemaVersionNumber', 'name', 'description', 'isSynchronized']
+    return CommandResults(
+        readable_output=tableToMarkdown(
+            f"Groups associated with user - {args.get('userName')} : ", response, headers=header_sequence,
+            headerTransform=pascalToSpace, removeNull=True),
+        outputs_prefix='STA.GROUP',
+        outputs_key_field=['id', 'name'],
+        outputs=response
+    )
+
```

Comment:
In all integration commands you should handle the case the response is empty. If the response is empty we dont want to create entry on context data and just return a CommandResult with human readable.

Reply:
Done. As discussed add below for commands.
 ```
   if not response:
        msg = 'No results found for the command.'
        return CommandResults(
            readable_output=msg,
        )
```